Tests/POEApi.Model.Tests/PoeModelTests.cs

```diff
@@ -205,6 +206,36 @@ public void GetShardCurrencyStashTest()
             }
         }
 
+        [TestMethod]
+        public void GetFragmentStashTest()
```

Comment:
The tooltip for splinters is incorrect, as they are displaying as a generic, style-less tooltip.  This is because the `Splinter` class is a subclass of `StackableItem` and there is not specific style for either of those types.  They should look like currency items, so either make `Splinter` a subclass of `Currency` (preferable) or add `Splinter` to the set of types that get a `CurrencyItemHoverViewModel` in `ItemHoverViewModelFactory.cs`.

(No good place for this comment, since neither of the files mentioned have been modified.)

Reply:
I'll have to leave this as I can't get your bin file to open. Could you raise a separate issue and I can try to resolve this as a separate PR.